pyswmm/links.py

```diff
@@ -94,14 +94,16 @@ def __getitem__(self, linkid):
     def __iter__(self):
         return self
 
-    def next(self):
+    def __next__(self):
         if self._cuindex < self._nLinks:
             linkobject = Link(self._model, self._linkid)
             self._cuindex += 1  # Next Iteration
             return linkobject
         else:
             raise StopIteration()
 
+    next = __next__  # Python 2
```

Comment:
So as I understand, the method `__next__` was day lighted in Python 3 [pep3114](https://www.python.org/dev/peps/pep-3114/).  In py2, by default, the iterator calls `next()` therefore, I had to add this line in order for it to step.  

I could use the `2to3` package but that doesn't feel like progress to me.  If you know of a better way, please enlighten me!